sample-apps/react/react-dogfood/components/MeetingUI.tsx

```diff
@@ -114,10 +114,7 @@ export const MeetingUI = ({ chatClient, mode }: MeetingUIProps) => {
 
   useEffect(() => {
     const handlePageLeave = async () => {
-      if (
-        call &&
-        [CallingState.JOINING, CallingState.JOINED].includes(callState)
-      ) {
+      if (call) {
         await call.leave();
       }
```

Comment:
Tangentially related to the goal of this PR, the call instance was not properly cleaned up in `react-dogfood` when navigating away from lobby.